config/config.go

```diff
@@ -91,6 +93,19 @@ type RedshiftConfig struct {
 	S3Region       string `toml:"-"`
 }
 
+type SnowflakeConfig struct {
+	Host           string
+	DB             string
+	User           string
+	Password       string
+	ExportTable    string
+	SyncTable      string
+	DatabaseSchema string
+	Credentials    string
+	VarCharMax     int
+	S3Region       string `toml:"-"`
+}
```

Comment:
This has the exact same structure as `Redshift`. I haven't worked with snowflake, but if this is all that's necessary, we should just merge these concepts into a single thing. Technically, the fact that it's labelled `Redshift` at all is misleading since it just assumes some database that supports the postgres driver.

Also, looking at `snowflake.go`, it's the same code as `redshift.go`. I'm curious, if you just plug in your values that you're using for snowflake the redshift config, does it just work? This would be great since it wouldn't require any changes to the source (aside from documentation, and possibly naming for clarity).